GetStream/stream-video-swift
# [Feature]VideoCodec preference

### 🔗 Issue Links

Resolves https://stream-io.atlassian.net/browse/PBE-6265

### 🎯 Goal

Provide a way for integrators, to select their preferred streaming codec.

### 🧪 Manual Testing Notes

- Select the preferred codec from the login screen > debug menu
- Join the call
- Ensure the selected codec is visible in the stats screen next to the Published Resolution stat

### ☑️ Contributor Checklist
